src/components/DashboardEditor/DashboardEditor.jsx

```diff
@@ -307,11 +323,16 @@ const DashboardEditor = ({
               setDashboardJson({
                 ...dashboardJson,
                 cards: dashboardJson.cards.map((card) =>
-                  card.id === cardConfig.id ? cardConfig : card
+                  card.id === cardConfig.id
```

Comment:
Perhaps the `onChange` could be a function with an explanatory name now since the inline function has grown larger

Reply:
Good call, I extracted it to `handleOnCardChange`